src/region/RegionAnnotations.tsx

```diff
@@ -76,34 +77,60 @@ export default class RegionAnnotations extends React.PureComponent<Props, State>
         createRegion({ ...staged, message });
     };
 
+    renderCreator = (): JSX.Element => {
+        const { isCreating, isRotated } = this.props;
+        const canCreate = isCreating && !isRotated;
+
+        return (
+            <>
+                {canCreate && (
+                    <RegionCreator
+                        className="ba-RegionAnnotations-creator"
+                        onStart={this.handleStart}
+                        onStop={this.handleStop}
+                    />
+                )}
+            </>
+        );
+    };
+
+    renderList = (): JSX.Element => {
+        const { activeAnnotationId, annotations } = this.props;
+
+        return (
+            <RegionList
+                activeId={activeAnnotationId}
+                annotations={annotations}
+                className="ba-RegionAnnotations-list"
+                onSelect={this.handleAnnotationActive}
+            />
+        );
+    };
+
     setRectRef = (rectRef: RegionRectRef): void => {
         this.setState({ rectRef });
     };
 
     render(): JSX.Element {
-        const { activeAnnotationId, annotations, isCreating, isRotated, message, staged, status } = this.props;
+        const { isCreating, isDiscoverabilityEnabled, isRotated, message, staged, status } = this.props;
         const { rectRef } = this.state;
         const canCreate = isCreating && !isRotated;
         const canReply = status !== CreatorStatus.started && status !== CreatorStatus.init;
         const isPending = status === CreatorStatus.pending;
 
         return (
             <>
-                {/* Layer 1: Saved annotations */}
-                <RegionList
-                    activeId={activeAnnotationId}
-                    annotations={annotations}
-                    className="ba-RegionAnnotations-list"
-                    onSelect={this.handleAnnotationActive}
-                />
-
-                {/* Layer 2: Drawn (unsaved) incomplete annotation target, if any */}
-                {canCreate && (
-                    <RegionCreator
-                        className="ba-RegionAnnotations-creator"
-                        onStart={this.handleStart}
-                        onStop={this.handleStop}
-                    />
+                {isDiscoverabilityEnabled ? (
+                    <>
+                        {/* With discoverability, put the RegionCreator below the RegionList so that existing regions can be clicked */}
```

Comment:
In the explicit region mode, we will not be able to create region annotation starting from the inside of another region annotation target. Is this the expected behavior?